cmd/micro/micro.go

```diff
@@ -29,6 +29,49 @@ import (
 	"github.com/zyedidia/tcell/v2"
 )
 
+const usage = `Usage: micro [OPTIONS] [FILE]...
+-clean
+    	Cleans the configuration directory
+-config-dir dir
+    	Specify a custom location for the configuration directory
+[FILE]:LINE:COL (if the ` + "`parsecursor`" + ` option is enabled)
```

Comment:
So this makes it not so unquestionable whether the code actually becomes more readable?